### ISSUE TITLE
Java crash when video capture is released

### ISSUE THREAD

**Issue by mdagraca** [openpnp/opencv#29]
Hi,  
I'm developing a system using your lib version 3.2.0-1, Java 1.8.0_51 and OSX 10.11.6 but I have an error when the video capture objects are release.

My code is:
 ```
 if (this.videoCapture.isOpened()) {
	this.videoCapture.release();
}
```
When the object is release java crash and a log is generated (attached)
[hs_err_pid13503.log](https://github.com/openpnp/opencv/files/1636796/hs_err_pid13503.log)

I replaced your lib with  opencv 3.3.1 jar and the same happens so I suppose that it's not related with your lib.
Which version of opencv are you using? Is the last one? if not, are you going to update it?
Do you have this release problem too? 
thanks in advantage
Mariela

**Comment by vonnieda** [comment]
Hi @mdagraca,

The OpenCV version is the one named in the jar version. So, 3.2.0-1 is OpenCV 3.2.0, jar version 1. Jar version 2 would be 3.2.0-2, and so on.

I do not run into this issue with my code, although I am using 2.4.11-2 in my project. Could you try with that version and see if you get the same thing?

For reference, the code I use is basically identical to yours: https://github.com/openpnp/openpnp/blob/develop/src/main/java/org/openpnp/machine/reference/camera/OpenCvCamera.java#L204

I have fallen behind on updating versions of this library. I'll put it on my schedule to get a new version pushed soon.

Thanks,
Jason

**Comment by mdagraca** [comment]
Thanks Jason.
I think there is something wrong with the threads. I'm testing it.
The videoCapture is opened when the user clicks on the start button (JavaFX) and released when the user clicks on the stop button.

**Comment by vonnieda** [comment]
I've just released version 3.4.2-0, please try this and see if it resolves the issue.

**Comment by vonnieda** [comment]
Closed due to no response. Please reopen if you are still having this issue.
